test/instrumentation/modules/redis.js

```diff
@@ -39,7 +41,7 @@ test(function (t) {
     t.end()
   })
 
-  var client = redis.createClient()
+  var client = redis.createClient('6379', host)
```

Comment:
If you change the argument to an object, you can get away with only supplying the hostname and if you don't need to provide a fallback to `localhost` either (since that's the default), so you can simplify it like so:

```js
redis.createClient({host: process.env.REDIS_HOST})
```